crypto-ffi/bindings/js/src/CoreCryptoInstance.ts

```diff
@@ -172,6 +172,16 @@ export function setMaxLogLevel(level: CoreCryptoLogLevel): void {
 export function buildMetadata(): CoreCryptoFfiTypes.BuildMetadata {
     return CoreCryptoFfi.build_metadata();
 }
+
+/**
+ * Returns the current version of {@link CoreCrypto}
+ *
+ * @returns The `core-crypto-ffi` version as defined in its `Cargo.toml` file
```

Comment:
This is public API so I wouldn't mention internal crate names and Rust specifics like `Cargo.toml`.
I rather suggest something like
```
 * @returns the CoreCrypto version as a string (e.g. "3.1.2")
```